src/pages/MovieDetail/MovieDetail.js

```diff
@@ -86,7 +85,7 @@ const MovieDetailPage = () => {
             draggable={false}
             showDots={true}
             responsive={responsive}
-            ssr={true} // means to render carousel on server-side.
+            ssr={true} 
```

Comment:
It is better to create new component ex. `carouselShowList`, you might use it in other pages